src/Statement.php

```diff
@@ -76,8 +78,9 @@
      * @param \DateTime|null    $stored
      * @param Context|null      $context
      * @param Attachment[]|null $attachments
+     * @param string            $version
      */
-    public function __construct(StatementId $id = null, Actor $actor, Verb $verb, Object $object, Result $result = null, Actor $authority = null, \DateTime $created = null, \DateTime $stored = null, Context $context = null, array $attachments = null)
+    public function __construct(StatementId $id = null, Actor $actor, Verb $verb, Object $object, Result $result = null, Actor $authority = null, \DateTime $created = null, \DateTime $stored = null, Context $context = null, array $attachments = null, $version = '1.0.3')
```

Comment:
Thanks for reading the spec so carefully. Fixed it and added a spec test to make sure we don't break this in the future.